### PR TITLE
Incident management feature updates

### PR DESCRIPTION
**Pull request by estherk15**
### What does this PR do?

Adds information on the following feature updates:
- Microsoft Teams and Zoom integrations
- Enable Private incidents and Delete in Incident Settings
- Timezone selector in Notifications for Incident Details
- Attribute Teams leverages Datadog Teams to add users to incident slack channel during an incident

### Motivation

[DOCS-5851](https://datadoghq.atlassian.net/browse/DOCS-5851)

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

[DOCS-5851]: https://datadoghq.atlassian.net/browse/DOCS-5851?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: content/en/service_management/incident_management/_index.md [hunk 1]
```diff
@@ -114,32 +112,20 @@ For more information about the Datadog Slack integration, check out [the docs][7
...
-**Additional notifications**: Notify other teams or people.
-
-Click on **Declare Incident** to finish creating your incident.
+| Incident elements    | Description |
+| ----------- | ----------- |
+| Title | (Required) Give your incident a descriptive title. |
+| Severity Level| (Required) Denotes the severity of your incident, from SEV-1 (most severe) to SEV-5 (least severe). If your incident is under initial investigation, and you do not know the severity yet, select UNKNOWN. <br> **Note**: You can customize the description of each severity level to fit the requirements of your organization.|
+| Incident Commander | This person is assigned as the leader of the incident investigation. |
+| Attributes (Teams) | Attach the appropriate group of users to an incident through [Datadog Teams][9], they are automatically invited to the Slack channels. |
+| Notifications | Specify a user, slack channel or external email to send notifications of this incident to.  |
```
Comment (drichards-87) [inline content/en/service_management/incident_management/_index.md:127]:
```suggestion
| Notifications | Specify a user, Slack channel or external email to send notifications of this incident to.  |
```

File: content/en/service_management/incident_management/_index.md [hunk 2]
```diff
@@ -114,32 +112,20 @@ For more information about the Datadog Slack integration, check out [the docs][7
...
-
-**Additional notifications**: Notify other teams or people.
-
-Click on **Declare Incident** to finish creating your incident.
+| Incident elements    | Description |
+| ----------- | ----------- |
+| Title | (Required) Give your incident a descriptive title. |
+| Severity Level| (Required) Denotes the severity of your incident, from SEV-1 (most severe) to SEV-5 (least severe). If your incident is under initial investigation, and you do not know the severity yet, select UNKNOWN. <br> **Note**: You can customize the description of each severity level to fit the requirements of your organization.|
+| Incident Commander | This person is assigned as the leader of the incident investigation. |
+| Attributes (Teams) | Attach the appropriate group of users to an incident through [Datadog Teams][9], they are automatically invited to the Slack channels. |
```
Comment (drichards-87) [inline content/en/service_management/incident_management/_index.md:126]: Should there be a word after the comma here? "Datadog Teams, they are"

File: content/en/service_management/incident_management/incident_details.md [hunk 3]
```diff
@@ -14,17 +14,21 @@ further_reading:
...
+## Overview
+
+{{< img src="/service_management/incidents/incident_details/incident_overview_page.png" alt="Incident details page of an Active SEV-4 incident." style="width:100%;">}}
+
 Every incident in Datadog has its own Incident Details page where you can manage your incident's property fields, signals, tasks, documents, responders, and notifications. An Incident Details page is available after you [create a new incident][1]. The Incident Details page contains a global header for quick access to key actions, while the remaining body of the page is divided into different sections using tabs to group related incident data together. The first of these sections is the Overview.
 
 ## Global header
 
-{{< img src="service_management/incidents/incident_global_header.jpeg" alt="Incident Global Header" style="width:80%;">}}
+{{< img src="/service_management/incidents/incident_details/global_header.png" alt="Global header on incident detail page showing assigned slack channel, CoScreen, Zoom, and Jira links" style="width:100%;" >}}
```
Comment (drichards-87) [inline content/en/service_management/incident_management/incident_details.md:25]:
```suggestion
{{< img src="/service_management/incidents/incident_details/global_header.png" alt="Global header on incident detail page showing assigned Slack channel, CoScreen, Zoom, and Jira links" style="width:100%;" >}}
```

File: content/en/service_management/incident_management/incident_details.md [hunk 4]
```diff
@@ -14,17 +14,21 @@ further_reading:
...
 
-The global header provides access to the [Status and Severity][2] selectors, and links to your [Incident Integrations][3]. After you've moved an incident to the resolved status, an option appears in the header to generate a postmortem Notebook using a [postmortem template][4]. Configure your postmortem templates in Incident Settings to predefine the structure and content of your postmortems.
+The global header provides access to the [Status and Severity][2] selectors, and links to your [Incident Integrations][3]. For more information on how to configure automatic Slack, Microsoft Teams, or Zoom links with every new incident, see [Incident Settings][4].
```
Comment (drichards-87) [inline content/en/service_management/incident_management/incident_details.md:27]:
```suggestion
The global header provides access to the [Status and Severity][2] selectors, and links to your [Incident Integrations][3]. For more information on how to configure automatic links with every new incident for Slack, Microsoft Teams, or Zoom, see [Incident Settings][4].
```

File: content/en/service_management/incident_management/incident_details.md [hunk 5]
```diff
@@ -42,7 +46,7 @@ Properties are divided into the following three sections:
 * Why it happened
 * Attributes
 
-In [Incident Settings][5], add additional property fields using the `<KEY>:<VALUE>` pairs from your Datadog metric tags, or create custom ones. Assign values to an incident's properties to easily search for a subset of incidents on the [Incident Homepage][6] and to form queries when using [Incident Management Analytics][7]. You can also reorder your property fields and move them to different headings so the most important properties are in prominent locations.
+In [Incident Settings][7], add additional property fields using the `<KEY>:<VALUE>` pairs from your Datadog metric tags, or create custom ones. Assign values to an incident's properties to easily search for a subset of incidents on the [Incident Homepage][8] and to form queries when using [Incident Management Analytics][9]. You can also reorder your property fields and move them to different headings so the most important properties are in prominent locations.
```
Comment (drichards-87) [inline content/en/service_management/incident_management/incident_details.md:49]:
```suggestion
In [Incident Settings][7], add additional property fields using the `<KEY>:<VALUE>` pairs from your Datadog metric tags, or create custom ones. Assign values to an incident's properties to enable you to search for a subset of incidents on the [Incident Homepage][8] and to form queries when using [Incident Management Analytics][9]. You can also reorder your property fields and move them to different headings so the most important properties are in prominent locations.
```

**Review by drichards-87 — APPROVED** [review]
Left you some light feedback and approved the PR.

File: content/en/service_management/incident_management/_index.md (hunk 2 above)
Comment (estherk15) [reply content/en/service_management/incident_management/_index.md:126]: @drichards-87 Datadog Teams automatically invites the members of the assigned Team to the slack channels. Does this read better:
```suggestion
| Attributes (Teams) | Assign [Datadog Teams][9] with the appropriate group of users to an incident, this ensures the relevant Teams are automatically invited to the Slack channels. |
```

Comment (drichards-87) [reply content/en/service_management/incident_management/_index.md:126]: How about something more along the lines of... "Assign the appropriate group of users to an incident using [Datadog Teams][9]. Members of the assigned team are automatically invited to the Slack channels."

Comment (estherk15) [reply content/en/service_management/incident_management/_index.md:126]:
```suggestion
| Attributes (Teams) | Assign the appropriate group of users to an incident using [Datadog Teams][9]. Members of the assigned team are automatically invited to the Slack channels. |
```

**Pull request merged**